src/app/zap-templates/zcl/data-model/chip/electrical-energy-measurement-cluster.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="Energy"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0091"/>
+    <field name="ImportedEnergy" mask="0x1"/>
+    <field name="ExportedEnergy" mask="0x2"/>
+    <field name="CumulativeEnergy" mask="0x4"/>
+    <field name="PeriodicEnergy" mask="0x8"/>
+  </bitmap>
+
+  <enum name="MeasurementTypeEnum" type="enum16">
+    <cluster code="0x0091"/>
+    <item name="Unspecified" value="0" />
+    <item name="Voltage" value="1" />
+    <item name="ActiveCurrent" value="2" />
+    <item name="ReactiveCurrent" value="3" />
+    <item name="ApparentCurrent" value="4" />
+    <item name="ActivePower" value="5" />
+    <item name="ReactivePower" value="6" />
+    <item name="ApparentPower" value="7" />
+    <item name="RMSVoltage" value="8" />
+    <item name="RMSCurrent" value="9" />
+    <item name="RMSPower" value="10" />
+    <item name="Frequency" value="11" />
+    <item name="PowerFactor" value="12" />
+    <item name="NeutralCurrent" value="13" />
+    <item name="ElectricalEnergy" value="14" />
+  </enum>
+
+  <struct name="EnergyMeasurementStruct">
+    <cluster code="0x0091"/>
+    <!-- TODO: Energy should be energy_mwh once supported -->
+    <item fieldId="0" name="Energy" type="int64s" />
```

Comment:
I know right now the type is int64s, but should be energy_mWh (but can't do that because of ZAP issues).
I have a similar PR which will fix this later - https://github.com/project-chip/connectedhomeip/pull/30701
Currently this uses energy_mwh (all lower case) but @bzbarsky-apple  has a set of ZAP changes which should resolve that in the coming days).

For now maybe we raise a new ticket to track the change?

Reply:
I cannot fix this in this PR, this will have to be fixed separately hence the TODO.